validations/src/validations.rs

```diff
@@ -1288,10 +1297,28 @@ pub fn update_utxo_diff(
     outputs: Vec<&ValueTransferOutput>,
     tx_hash: Hash,
 ) {
+    let mut input_pkh = if inputs.is_empty() {
+        None
+    } else {
+        let o = inputs.first().unwrap().output_pointer();
```

Comment:
Instead of using unwrap, you can use pattern matching. So instead of `if inputs.is_empty()`, do `if let Some(first) = inputs.first()`